src/main/java/frc/robot/stateMachine/StateMachine.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.stateMachine;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class StateMachine {
+    private ArrayList<Path> paths;
```

Comment:
To implement the suggestion about Edges, you could have a HashMap here with keys being the state names and the values being ArrayLists of edges